courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/tensors-variables.ipynb

```diff
@@ -19,7 +19,7 @@
     "\n",
```

Comment:
<p>Change to TODO 3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Hi @gwendolynstripling I have updated the sequencing of TODOs, feel free to merge the PR.
Thanks!